### DISCUSSION TITLE
Can I do this?

### DISCUSSION THREAD

**Discussion by hniu-tamu** [hniu-tamu/GEOG-510#2] [Q&A]
How can I install geemap?

**Comment by hniu-tamu** [comment]
Of course

**Comment by ccalvocm** [comment]
To install geemap, you can use pip, which is a package installer for Python. Here are the steps to install geemap:

Open a command prompt or terminal window.
Type pip install geemap and press Enter.
Wait for the installation to complete. You should see a message that says "Successfully installed geemap" when it's done.

Once geemap is installed, you can import it in your Python code using import geemap.
